projects/v2/src/app/services/sheet.service.ts

```diff
@@ -81,8 +81,8 @@ export class SheetService {
         ontologyCode,
         desc: res.description,
         iri: res.link,
-        entrezLink: res.extraLinks["Entrez Link"],
-        uniprotLink: res.extraLinks["Uniprot Link"],
+        entrezLink: res.extraLinks['Entrez Link'],
```

Comment:
Should just need to track 'extraLinks', not hard code uniprot + entrez

Reply:
ok Bruce. Got it. Let me do the necessary changes.